libtextsecure/sendmessage.js

```diff
@@ -869,9 +873,7 @@ MessageSender.prototype = {
   },
 
   sendGroupProto(providedNumbers, proto, timestamp = Date.now(), options = {}) {
-    const me = textsecure.storage.user.getNumber();
-    const numbers = providedNumbers.filter(number => number !== me);
```

Comment:
is it intended to remove "remove ourselves" from when there's no provided numbers

Reply:
Sorry i didn't get this question 🤔

Reply:
This removed code, filters out ourself from the return (if no provided numbers). Did you mean to change this behavior?

Reply:
Yes, we need to send our number to `sendMessageProto` to trigger a sync message to be sent